plugins/modules/foreman_scc_account.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2019 Manisha Singhal (ATIX AG)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: foreman_scc_account
+short_description: Manage Foreman SccAccount
+description:
+  - "Manage Foreman SccAccount Entities"
+  - "This module requires the foreman_scc_manager plugin set up in the server"
+  - "See: U(https://github.com/ATIX-AG/foreman_scc_manager)"
+author:
+  - "Manisha Singhal (@manisha15) ATIX AG"
+options:
+  name:
+    description: Name of the scc_account
+    required: true
+    type: str
+  login:
+    description: Login id of scc_account
+    required: true
+    type: str
+  scc_account_password:
+    description: Password of scc_account
+    required: true
+    type: str
+  base_url:
+    description: URL of SUSE for scc_account
+    required: true
+    type: str
+  interval:
+    description: Interval for syncing scc_account
+    required: false
+    type: str
+    choices: ["never", "daily", "weekly", "monthly"]
+  sync_date:
+    description: Last Sync time of scc_account
+    required: false
+    type: str
+  organization:
+    description: Name of related organization
+    type: str
+  test_connection:
+    description: Test connection of scc_account
```

Comment:
Can you be a bit more specific here?
We should say, that this will add a safeguard test to not accidentially change the accounts credentials to something that is not able to connect to the server.